crates/primitives-traits/src/node.rs

```diff
@@ -0,0 +1,37 @@
+use crate::{BlockBody, FullBlock, FullReceipt, FullSignedTx};
+use core::fmt;
```

Comment:
```suggestion
use core::fmt;

use crate::{BlockBody, FullBlock, FullReceipt, FullSignedTx};
```
a bit easier to read, for example wrt std/no-std support, when ordered after generality:
```
std and std equivalents core and alloc

deps from outside crate

crate and super
```